asr-python-adapter-finetune-am-conformer-nemo.ipynb

```diff
@@ -0,0 +1,186 @@
+    "\n",
+    "Another minor difference is the use of spectrogram augmentation (SpecAugment). During fine-tuning, the amount of data is very little compared to when the model was pre-trained with thousands of hours of speech, so SpecAugment may have been used to prevent overfitting. For adapters, we recommend first disabling spec augmentation and trying to see if results are good enough, then adding it back slowly as needed to see if any improvements can be obtained.\n",
+    "\n",
+    "Please refer to the **Setup training and evaluation of the model** section, and particularly the **Setup Spectrogram Augmentation** section of the [NeMo Tutorial](#asr-with-adapters) for further details."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating and Training an Adapter\n",
+    "\n",
+    "The **Adapters: Creation and Preparation** section of the [NeMo Tutorial](#asr-with-adapters) details how to create an config object for a **LinearAdapter**, add it to the pretrained model, then freeze the original parameters of the model to train just the new parameters.\n",
+    "\n",
+    "All of this is a few lines of code, which will initialize your model so that it is prepared to train the adapter on new speech data."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluating the model\n",
+    "\n",
+    "Since adapters contain new parameters and are dynamic in nature, they can be enabled/disabled prior to evaluation. The **Evaluate the adapted model** section in the [NeMo Tutorial](#asr-with-adapters) showcases how you can load the model, enable or disable an adapter in the model, save the model and then evaluate it with some data. \n",
+    "\n",
+    "Adapters are trained such that if one disables all adapters, the model will revert back to the pretrained model, which allows you to select whether you want to do well on general speech (with original pretrained model) or do better on your adaptation domain (with an adapter enabled)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Export and Use in RIVA\n",
+    "\n",
+    "The nemo2riva command-line tool provides the capability to export your .nemo model in a format that can be deployed using NVIDIA Riva, a highly performant application framework for multi-modal conversational AI services using GPUs. A Python .whl file for nemo2riva is included in the Riva Quick Start resource folder. You can also install nemo2riva with pip, as shown in the Conformer-CTC fine-tuning tutorial.\n",
```

Comment:
```suggestion
    "The `nemo2riva` command-line tool provides the capability to export your `.nemo` model in a format that can be deployed using NVIDIA Riva, a highly performant application framework for multi-modal conversational AI services using GPUs. A Python `.whl` file for `nemo2riva` is included in the Riva Quick Start resource folder. You can also install `nemo2riva` with `pip`, as shown in the Conformer-CTC fine-tuning tutorial.\n",
```

Reply:
Can we make "Conformer-CTC fine-tuning tutorial" a link?

Reply:
Yup, I've added the link.